### DISCUSSION TITLE
Error stepwise RM

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#847] [Q&A] [answered]
Bagaimana mengatasi eeror di bawah ini 

![Screenshot (67)](https://github.com/teamalgoritma/community/assets/150925524/62634468-4044-4014-be9b-264489ca8c8d)

**Comment by hwulanayu** [comment, marked as answer]
Dear @rastitiratnasari,

Error yang Anda lampirkan sepertinya sama dengan error yang ada pada [stackoverflow](https://stackoverflow.com/questions/55342788/error-in-lengthobj-class-name-too-long-in-length) ini. Cobta tambahkan nama library sebelum fungsi step(). Contoh:

`startup_back <- stats::step(startup_all, direction = "backward")` 

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Reply by IrfanChairurrachman** [reply]
Halo @rastitiratnasari, karena tidak ada konfirmasi lebih lanjut dan untuk menjaga agar forum diskusi tetap berjalan kondusif, kami akan melakukan lock/close diskusi ini. Semoga dapat dipahami.
